module/actor/sheets/WitcherActorSheet.js

```diff
@@ -1,240 +1,238 @@
-import { extendedRoll } from "../../scripts/rolls/extendedRoll.js";
-import { RollConfig } from "../../scripts/rollConfig.js";
-import { ExecuteDefense } from "../../scripts/combat/defenses.js";
+import { extendedRoll } from "../../scripts/rolls/extendedRoll.js"
+import { RollConfig } from "../../scripts/rollConfig.js"
+import { ExecuteDefense } from "../../scripts/combat/defenses.js"
 
 import { sanitizeMixin } from "./mixins/sanitizeMixin.js"
-import { deathsaveMixin } from "./mixins/deathSaveMixin.js";
-import { criticalWoundMixin } from "./mixins/criticalWoundMixin.js";
-import { noteMixin } from "./mixins/noteMixin.js";
-import { globalModifierMixin } from "./mixins/globalModifierMixin.js";
-import { skillModifierMixin } from "./mixins/skillModifierMixin.js";
-import { skillMixin } from "./mixins/skillMixin.js";
-import { statMixin } from "./mixins/statMixin.js";
-import { itemMixin } from "./mixins/itemMixin.js";
+import { deathsaveMixin } from "./mixins/deathSaveMixin.js"
+import { criticalWoundMixin } from "./mixins/criticalWoundMixin.js"
+import { noteMixin } from "./mixins/noteMixin.js"
+import { globalModifierMixin } from "./mixins/globalModifierMixin.js"
+import { skillModifierMixin } from "./mixins/skillModifierMixin.js"
+import { skillMixin } from "./mixins/skillMixin.js"
+import { statMixin } from "./mixins/statMixin.js"
+import { itemMixin } from "./mixins/itemMixin.js"
 
-import { itemContextMenu } from "./interactions/itemContextMenu.js";
+import { itemContextMenu } from "./interactions/itemContextMenu.js"
```

Comment:
You are removing ;

Reply:
Hmm, maybe this ia my editor remove ; i will fix that too